### ISSUE TITLE
Developer Portal primary navigation missing nav element

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#10185]
### Description:

1. Left navigation should come in a `nav` element  
2. have the role defined `role="navigation"`
3. Set area label to main navigation `aria-label="Main navigation"`

![image](https://user-images.githubusercontent.com/20179540/110296844-cc4b7600-8018-11eb-9326-d62a283e45dd.png)

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---
